katago_server/trainings/models/network.py

```diff
@@ -118,4 +121,8 @@ def save(self, *args, **kwargs):
             # TODO: We should let the api to create a model throw if there is no parent mode
             if not self.parent_network:
                 self.parent_network = Network.objects.last()
+        # Allow blank file only if random
+        if (not self.is_random) and (self.model_file is None or len(self.model_file) <= 0):
```

Comment:
I prefer extracting the condition and removing the comment:

```python
is_initial_random_network = (not self.is_random) and (self.model_file is None or len(self.model_file) <= 0)
if not is_initial_random_network:
  ...
```

Reply:
ok